scripts/test/runtests.sh

```diff
@@ -81,4 +85,7 @@ done
 popd
 set -e
 
+#resetting the path so that we can be good neighbors to other scripts running after this.
+export PATH=$StartPath
```

Comment:
Only scripts which are started by this script or source this script (which they shouldn't be) scripts which execute this script will be unaffected